app/javascript/lib/vue-components/modules/BlockHeader.vue

```diff
@@ -32,16 +31,16 @@ export default {
     return {
       labelAll: "",
       labelNone: ""
-    }
+    };
   },
   created() {
-    this.labelAll = I18n.t("gobierto_investments.projects.all")
-    this.labelNone = I18n.t("gobierto_investments.projects.none")
+    this.labelAll = I18n.t("gobierto_common.vue_components.block_header.all");
```

Comment:
What about renaming `vue_components` to `js_components`?

Reply:
You mean in every place, right? Both translations and folder?

Reply:
I mean the key in the translation, but you are right the folder is also vue-component, so leave it.